docker/service/Dockerfile

```diff
@@ -1,5 +1,21 @@
-FROM adoptopenjdk/openjdk11:slim
+FROM gradle:jdk14 AS build
 
-COPY gp2gp.jar /
+COPY --chown=gradle:gradle ./service /home/gradle/service
 
-ENTRYPOINT java -jar /gp2gp.jar --port 8080
+WORKDIR /home/gradle/service
+
+RUN ./gradlew check
+
+RUN ./gradlew test --build-cache bootJar
+
+FROM adoptopenjdk/openjdk14:jre
```

Comment:
We're using jdk11 (latest LTS) - jdk14 is already out of support

Reply:
I would not be so sure about this - https://github.com/nhsconnect/integration-adaptor-111/blob/master/Dockerfile.111#L6